### PR TITLE
Port Parkstation Benches

### PR DESCRIPTION
**Pull request by rosieposieeee**
## About the PR

image

Motha freakin benches baby.

Comes with all the parkstation ones that are also on frontier.

The corner sofa doesn't buckle but otherwise they're functional. Should spice up mapping a bunch.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added benches, sofas, and pews!

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Parkstation/benches.yml
Comment (FluffiestFloof) [inline Resources/Prototypes/DeltaV/Parkstation/benches.yml:1]: This is looks like a duplicate of Resources/Prototypes/SimpleStation14/benches.yml

Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Parkstation/benches.yml:1]: I contractually have to fuck up one thing with every PR I ever send. Sorry.

**Review by DebugOk — DISMISSED** [review]
SimpleStation14 would be yet another addition namespace that has to be watched. I'm going to need to think on that first before looking into this.

Marked as requested changes to blocked merging

**Comment by rosieposieeee** [comment]
> SimpleStation14 would be yet another addition namespace that has to be watched. I'm going to need to think on that first before looking into this.
> 
> Marked as requested changes to blocked merging

Would it be better if i just moved these files into a DeltaV version?

**Comment by DEATHB4DEFEAT** [comment]
> > SimpleStation14 would be yet another addition namespace that has to be watched. I'm going to need to think on that first before looking into this.
> > Marked as requested changes to blocked merging
> 
> Would it be better if i just moved these files into a DeltaV version?

it's clearer that it was originally SimpleStation14's if in the SimpleStation14 namespace, and some may look at it as you claiming it as yours
Nyano took the Corvax round notifications and put it in the Nyano namespace (IIRC without making it clear where they got it from originally), which I complained about

**Comment by DebugOk** [comment]
> > > SimpleStation14 would be yet another addition namespace that has to be watched. I'm going to need to think on that first before looking into this.
> > > Marked as requested changes to blocked merging
> > 
> > 
> > Would it be better if i just moved these files into a DeltaV version?
> 
> it's clearer that it was originally SimpleStation14's if in the SimpleStation14 namespace, and some may look at it as you claiming it as yours Nyano took the Corvax round notifications and put it in the Nyano namespace (IIRC without making it clear where they got it from originally), which I complained about

I agree, it'd be best to leave them as is

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/parkbench_bamboo.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "TGStation",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/parkbench_bamboo.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from TGStation at https://github.com/lessthnthree/tgstation/blob/2ca948b578cdb46b60d19e029782483b1218004e/icons/obj/sofa.dmi",
```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/parkbench_bamboo.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "right",
+      "directions": 4
+    },
+    {
+      "name": "middle",
+      "directions": 4
+    }
+  ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/parkbench_bamboo.rsi/meta.json:23]:
```suggestion
}

```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/parkbench_wooden.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "TGStation",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/parkbench_wooden.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from TGStation at https://github.com/lessthnthree/tgstation/blob/2ca948b578cdb46b60d19e029782483b1218004e/icons/obj/sofa.dmi",
```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/parkbench_wooden.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "right",
+      "directions": 4
+    },
+    {
+      "name": "middle",
+      "directions": 4
+    }
+  ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/parkbench_wooden.rsi/meta.json:23]:
```suggestion
}

```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/pews.rsi/meta.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "TGStation",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/pews.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from TGStation at https://github.com/lessthnthree/tgstation/blob/2ca948b578cdb46b60d19e029782483b1218004e/icons/obj/sofa.dmi",
```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/pews.rsi/meta.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      "name": "arm_left",
+      "directions": 4
+    },
+    {
+      "name": "arm_right",
+      "directions": 4
+    }
+  ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/pews.rsi/meta.json:31]:
```suggestion
}

```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/sofa.rsi/meta.json [hunk 7]
```diff
@@ -0,0 +1,35 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "TGStation",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/sofa.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from TGStation at https://github.com/lessthnthree/tgstation/blob/2ca948b578cdb46b60d19e029782483b1218004e/icons/obj/sofa.dmi",
```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/sofa.rsi/meta.json [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    {
+      "name": "arm_left",
+      "directions": 4
+    },
+    {
+      "name": "arm_right",
+      "directions": 4
+    }
+  ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/sofa.rsi/meta.json:35]:
```suggestion
}

```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/sofa_corp.rsi/meta.json [hunk 9]
```diff
@@ -0,0 +1,35 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "TGStation",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/sofa_corp.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from TGStation at https://github.com/lessthnthree/tgstation/blob/2ca948b578cdb46b60d19e029782483b1218004e/icons/obj/sofa.dmi",
```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/sofa_corp.rsi/meta.json [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    {
+      "name": "arm_left",
+      "directions": 4
+    },
+    {
+      "name": "arm_right",
+      "directions": 4
+    }
+  ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/sofa_corp.rsi/meta.json:35]:
```suggestion
}

```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/steel_bench.rsi/meta.json [hunk 11]
```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "TGStation",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/steel_bench.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from TGStation at https://github.com/lessthnthree/tgstation/blob/2ca948b578cdb46b60d19e029782483b1218004e/icons/obj/sofa.dmi",
```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/steel_bench.rsi/meta.json [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "right",
+      "directions": 4
+    },
+    {
+      "name": "middle",
+      "directions": 4
+    }
+  ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/steel_bench.rsi/meta.json:23]:
```suggestion
}

```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/steel_bench_white.rsi/meta.json [hunk 13]
```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "TGStation",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/steel_bench_white.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from TGStation at https://github.com/lessthnthree/tgstation/blob/2ca948b578cdb46b60d19e029782483b1218004e/icons/obj/sofa.dmi",
```

File: Resources/Textures/SimpleStation14/Structures/Furniture/Benches/steel_bench_white.rsi/meta.json [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "right",
+      "directions": 4
+    },
+    {
+      "name": "middle",
+      "directions": 4
+    }
+  ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Structures/Furniture/Benches/steel_bench_white.rsi/meta.json:23]:
```suggestion
}

```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]
Psp was lazy with the sources

**Comment by DEATHB4DEFEAT** [comment]
guh
All my suggestions done in separate commits
You can go to the files tab and add them to a batch, then commit

**Comment by DebugOk** [comment]
> guh All my suggestions done in separate commits You can go to the files tab and add them to a batch, then commit

Our poor workflow runs. Thank god they're free. This'll probably cause a rate limit though

**Comment by DEATHB4DEFEAT** [comment]
> > guh All my suggestions done in separate commits You can go to the files tab and add them to a batch, then commit
> 
> Our poor workflow runs. Thank god they're free. This'll probably cause a rate limit though

Oh god, yeah

**Pull request closed without merging**